boards/arm/mps3_an547/mps3_an547.dts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2018-2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <arm/armv8-m.dtsi>
```

Comment:
Should we then have an armv8_1-m.dtsi header? Makes sense to me since we introduce separate Kconfig options.

Reply:
Will add various v8.1m dtsi/bindings and such.